src/frontend/src/test-e2e/selenium.test.ts

```diff
@@ -244,7 +244,7 @@ test("Log into client application, after registration", async () => {
 
     // default value
     const exp = await browser.$("#expiration").getText();
-    expect(Number(exp) / (24 * 60 * 60_000_000_000)).toBeCloseTo(1);
+    expect(Number(exp) / (8 * 60 * 60_000_000_000)).toBeCloseTo(1);
```

Comment:
what's this? updated version of something?

Reply:
Yes, the updated Agent-js changed the session validity default.